pkg/store/registry.go

```diff
@@ -58,6 +58,18 @@ func NewStoreRegistry(config *StoresConfig) (StoreRegistry, error) {
 			}
 			registry[key] = store
 
+		case "azure-key-vault":
+			var opts AzureKeyVaultStoreOptions
+			if err := parseOptions(storeConfig.Options, &opts); err != nil {
+				return nil, fmt.Errorf("failed to parse Key Vault store options: %w", err)
+			}
```

Comment:
Please keep the case statement in alphabetical order (so move this block up with the other `a` stores.